### PR TITLE
cldr-emoji-annotation: new recipe

### PR DESCRIPTION
**Pull request by pengphei**
Signed-off-by: Han Pengfei <pengphei@qq.com>

### CODE REVIEW

File: app-i18n/cldr-emoji-annotation/cldr_emoji_annotation-37.0_13.0_0_2.recipe [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+HOMEPAGE="https://github.com/fujiwarat/cldr-emoji-annotation"
+COPYRIGHT="1991-2022, Unicode, Inc"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/fujiwarat/cldr-emoji-annotation/archive/${portVersion}/cldr-emoji-annotation-$portVersion.tar.gz"
+CHECKSUM_SHA256="22fbf6a96f6966edaff2bd23694568da394e064a0599ef5c6047fffe233cab16"
+SOURCE_DIR="cldr-emoji-annotation-$portVersion"
+
+ARCHITECTURES="any"
+SECONDARY_ARCHITECTURES="x86"
```
Comment (Begasus) [inline app-i18n/cldr-emoji-annotation/cldr_emoji_annotation-37.0_13.0_0_2.recipe:13]: No need for SECONDARY_ARCHITECTURES

File: app-i18n/cldr-emoji-annotation/cldr_emoji_annotation-37.0_13.0_0_2.recipe [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	cmd:autoreconf
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	./autogen.sh
```
Comment (Begasus) [inline app-i18n/cldr-emoji-annotation/cldr_emoji_annotation-37.0_13.0_0_2.recipe:36]: autogen.sh calls for aclocal, autoheader, autoconf and automake, it also calls configure, so maybe just drop autogen.sh here and run autoreconf -vfi ?

**Review by Begasus — CHANGES_REQUESTED** [review]

**Comment by korli** [comment]
Why use this package? From the README: 
> I decided to use tarballs in https://github.com/unicode-org/cldr/releases
directly and this repo is no longer maintained.

**Comment by pengphei** [comment]
> Why use this package? From the README:
> 
> > I decided to use tarballs in https://github.com/unicode-org/cldr/releases
> > directly and this repo is no longer maintained.

hmmm, I have checked opensuse, which uses cldr-emoji-annotation instead of unicode-cldr.

https://build.opensuse.org/package/show/M17N/cldr-emoji-annotation.

well, gentoo use unicode-cldr.

https://github.com/gentoo/gentoo/tree/master/app-i18n/unicode-cldr

I haven't figured out they provide the same thing ;(

**Pull request closed without merging**